lasagna-master/hemartin/lasagna_master.go

```diff
@@ -0,0 +1,70 @@
+package lasagna
+
+const noodleQuantityPerLayer int = 50
+const sauceQuantityPerLayer float64 = 0.2
+const originalQuantityPortions float64 = 2.0
+
+func PreparationTime(layers []string, layerPrepTime int) int {
+	if layerPrepTime <= 0 {
+		layerPrepTime = 2
+	}
+	return layerPrepTime * len(layers)
+
+}
+
+func Quantities(layers []string) (int, float64) {
+	return QuantitiesFast(layers)
+}
+
+// This was the first implementation
+func QuantitiesSlow(layers []string) (int, float64) {
```

Comment:
Is the slower performance of this function because of the lookup time for the map or is it something else?

Reply:
Yeah I think it is slower to access the map than sum floats directly.